src/System.Runtime/src/System.Runtime.builds

```diff
@@ -2,9 +2,15 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Runtime.CoreCLR.csproj">
+    <Project Include="System.Runtime.csproj">
       <AdditionalProperties>TargetGroup=</AdditionalProperties>
     </Project>
+    <Project Include="System.Runtime.csproj">
```

Comment:
I think we want to include these only on Windows, i.e.

```
    <Project Include="System.Runtime.csproj" Condition="'$(TargetsWindows)' == 'true'">
      <AdditionalProperties>TargetGroup=netcore50aot</AdditionalProperties>
    </Project>
```
Same for the net46 group.